sites/partners/src/listings/PaperListingForm/sections/ApplicationDates.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import { useFormContext } from "react-hook-form"
+import { t, GridSection, DateField, TimeField } from "@bloom-housing/ui-components"
+import { FormListing } from "../index"
+import moment from "moment"
+
+type ApplicationDatesProps = {
+  listing?: FormListing
+}
+
+const ApplicationDates = ({ listing }: ApplicationDatesProps) => {
+  const formMethods = useFormContext()
```

Comment:
hmm, not sure, but probably we have an access to the form data from the `useFormContext()`?
If yes - the additional prop `listing` is not necessary :)

`const { register, watch, formState } = formMethods`

https://react-hook-form.com/api/useform/formstate

Reply:
I don't see the listing here as part of form state when I restructure it and log it in the console as a heads up

Reply:
getValues should have it.

Reply:
so getValues has the form fields but the form fields don't match directly to the listing, so I can get `applicationDueTimeField: { hours: "",  minutes: "" , period: "am" , seconds: ""  }` but not the actual time, for now I'm still passing the listing as a prop but let me know if you'd prefer a different approach

Reply:
I'm ok with this for now. I think at some point we may want to revisit the date and times fields to work in a way that we don't have to specify a separate "field', as in `applicationDueDateField` vs `applicationDueDate`, though I understand the way it is now makes sense, because it doesn't tie a date library to the component.